templates/CMakeLists.txt.template

```diff
@@ -356,18 +361,18 @@
   % for tgt in targets:
   % if tgt.build in ["all", "protoc", "tool", "test", "private"] and not tgt.boringssl:
   % if tgt.build in ["test", "private"]:
-  if (gRPC_BUILD_TESTS)
+  if(gRPC_BUILD_TESTS)
```

Comment:
was removing the space intended? If not, let's stay consistent with the rest of the file.

Reply:
Yes, that was intentional.

Reply:
It the original formatting against cmake code style? If so, can we do things like updating formatting in a separate PR (and do that consistently everywhere, if it's important).

Reply:
It is indeed not the recommended style, though obviously there are many projects that use it. I thought it was a mistake and did not realize it was intentional. However, duplicating the condition in the `endif()` block is even *more* not-recommended... is it OK if I at least get rid of that for this particular `if()` block?

Reply:
sg, but please file an issue with the list of cmake style problems that we currently have in our CMakeLists.txt?
also, is there some good linter for cmake files?

Reply:
> also, is there some good linter for cmake files?

Not that I know of :( we've talked about writing one, but I don't know if any real progress has been made on it.